lib/normalize_title_filter.js

```diff
@@ -43,7 +43,8 @@ module.exports = function(hyper, req, next, options, specInfo) {
             .then(function(result) {
                 var resultText = result.getPrefixedDBKey();
                 if (resultText !== rp.title) {
-                    if (result.getNamespace().isUser() || result.getNamespace().isUserTalk()) {
+                    if (result.getNamespace().isUser()
+                            || result.getNamespace().isUserTalk()) {
```

Comment:
Can we fold POST into this? Explicitly disabling caching for that case would be a good idea as well.